### PR TITLE
OpenStack: Image name validations

### PR DESCRIPTION
**Pull request by Fedosin**
When the image is provided by user, i.e. not created automatically by the installer, we need to check that the image exists and there are no other images with this name.

### CODE REVIEW

File: pkg/rhcos/openstack.go [hunk 1]
```diff
@@ -41,3 +43,39 @@ func GenerateOpenStackImageName(rhcosImage, infraID string) (imageName string, i
 
 	return infraID + "-rhcos", true
 }
+
+// ValidateOverriddenOpenStackImageName checks if an overridden image name exists and there are no other images with this name.
+func ValidateOverriddenOpenStackImageName(imageName, cloud string) error {
```
Comment (abhinavdahiya) [inline pkg/rhcos/openstack.go]: please move this to `pkg/tfvars/openstack`, that seems like the best place for this.

Also would like to recommend this be private function and not part of the package API

Comment (Fedosin) [reply pkg/rhcos/openstack.go]: Thanks! done

**Pull request merged**